circle.yml

```diff
@@ -7,6 +7,12 @@ jobs:
     working_directory: /go/src/github.com/pingcap/parser
     steps:
       - checkout
+      - run:
+          name: "Verify parser.go is up-to-date"
+          command: |
+            mv parser.go parser.go.committed
+            make parser
+            diff -u parser.go.committed parser.go
```

Comment:
What will happen in the CI if parser.go is out of date?

Reply:
It's also possible to use git commands without moving files. To ensure no caching you can rm the file

Reply:
@shenli it will look like this: https://circleci.com/gh/pingcap/parser/3

@gregwebs Yes we could do

```sh
rm parser.go
make parser
git diff --exit-code
```

but I don't see too much advantages? 😕